PerimeterXModule/Internals/Helpers/PxConstants.cs

```diff
@@ -22,7 +22,7 @@ public static class PxConstants
 
 		// Endpoints
 		public const string RISK_API_V2 = "/api/v2/risk";
-		public const string CAPTCHA_API_V1 = "/api/v1/risk/captcha";
+		public const string CAPTCHA_API_V2 = "/api/v2/risk/captcha";
```

Comment:
this const name pretty much misses the concept of a constant, since you have to change any location that uses it. should be changed to `CAPTCHA_API_PATH` or something like that.

Reply:
Agree. Will change.